### ISSUE TITLE
MUSR: zero field controller mode reverts back to manual mode on config change

### ISSUE THREAD

**Issue by rerpha** [ISISComputingGroup/IBEX#6360]
As a MuSR instrument scientist I would like the ZF mode (auto/manual) to stay consistent when changing configuration. the IS for MuSR believes EMU may already have this and would like the same behaviour on both instruments as this could confuse users if they thought the ZF system was in auto feedback mode in a previous run but switched unknowingly on config change. 

### Acceptance criteria
- ZF system on MuSR stays in the same state between config changes and doesn't revert back to manual if in auto feedback mode

**Comment by rerpha** [comment]
PR: https://github.com/ISISComputingGroup/EPICS-ZFCNTRL/pull/10

Release notes: https://github.com/ISISComputingGroup/IBEX/pull/6362

**Comment by rerpha** [comment]
I am going to change this and make it safer (for instruments other than MuSR and possibly EMU) by using a macro which will turn autosave on and off. This will need to be set in globals.txt rather than in every config.

**Comment by rerpha** [comment]
I am going to hotfix this on MuSR but not on any other instruments, PINI is already set to yes in the ZFCTRL IOC so don't want to break it by setting macros etc.

**Comment by rerpha** [comment]
Have hotfixed on MuSR. To review: check hotfix looks OK, check hotfix list has been updated. I will email the instrument scientist.

**Comment by rerpha** [comment]
seems like its not quite working , will spend some more time figuring out after testing the release

**Comment by rerpha** [comment]
trying this: 

```
record(bo, "$(P)AUTOFEEDBACK") {
  field(DESC, "Mode of ZF system")
  field(ZNAM, "Manual")
  field(ONAM, "Auto-feedback")
  
  # There is a requirement to force manual mode on IOC start.
  # field(VAL, "0")
  field(PINI, "YES")
  
  info(interest, "HIGH")
  info(archive, "5.0 VAL")
  info(autosaveFields, "VAL")
  
}
```

works locally, i hotfixed this to MuSR but unable to test it out now as they are soak testing. Have asked the IS if I can test it out on the instrument machine so will come back to this when I can.

**Comment by rerpha** [comment]
Appears to be working now, just restarted the IOC while it was in auto-feedback mode and it came back in auto-feedback mode.

**Comment by rerpha** [comment]
Have spun out a more permanent fix to #6383
